lstchain/scripts/lstchain_add_source_dependent_parameters.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
 
 """
-Script to add the source dependent parameters to a DL1 file
+Script to add the source dependent parameters to a DL1 file.
 
-- Input: DL1 data file.
-- Output: DL1 data file.
+- Input: DL1 data file. The program returns the same DL1 data file with
```

Comment:
It does not return. It modifies in place. 

Maybe: `Input: DL1 data file. Source dependent parameters will be added to this file.`

Reply:
done